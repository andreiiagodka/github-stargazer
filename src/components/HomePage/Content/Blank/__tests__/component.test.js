import { shallow } from 'enzyme'

import BlankComponent from '..'

describe('<BlankComponent />', () => {
  it('renders correctly', () => {
    const wrapper = shallow(<BlankComponent />)

    expect(wrapper).toMatchSnapshot()
  })
})
