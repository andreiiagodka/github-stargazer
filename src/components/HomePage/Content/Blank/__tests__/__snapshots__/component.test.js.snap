// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<BlankComponent /> renders correctly 1`] = `
<Row
  noGutters={false}
>
  <Col
    className="text-center"
    md={12}
  >
    <p
      className="lead"
    >
      No repositories added
    </p>
  </Col>
</Row>
`;
